src/app-layer-enip-common.h

```diff
@@ -24,10 +24,7 @@
 #ifndef __APP_LAYER_ENIP_COMMON_H__
 #define __APP_LAYER_ENIP_COMMON_H__
 
-#include "app-layer-protos.h"
-#include "app-layer-parser.h"
-#include "flow.h"
-#include "queue.h"
+#include "rust.h"
```

Comment:
it does feel a bit weird to include `rust.h` here when the parser doesn't use rust

Reply:
But it does : `AppLayerTxData` is defined in rust